setup.py

```diff
@@ -49,7 +49,7 @@
 def get_download_url() -> tuple[str, str]:
     postfix, sha256 = POSTFIX_SHA256[(sys.platform, platform.machine())]
     url = (
-        f'https://github.com/koalaman/shellcheck/releases/download/'
+        'https://github.com/koalaman/shellcheck/releases/download/'
```

Comment:
Fell free to run this command in order to recognize that the change to be introduce do not affect the output:

```python
SHELLCHECK_VERSION: str = '0.9.0'


if __name__ == '__main__':

    postfix: str ='linux'

    # @info: The f-string on the next fiew lines is non-useful.
    url = (
        f'https://github.com/koalaman/shellcheck/releases/download/'
        f'v{SHELLCHECK_VERSION}/shellcheck-v{SHELLCHECK_VERSION}.{postfix}'
    )

    assert 'https://github.com/koalaman/shellcheck/releases/download/v0.9.0/shellcheck-v0.9.0.linux' == url

    url = (
        'https://github.com/koalaman/shellcheck/releases/download/'
        f'v{SHELLCHECK_VERSION}/shellcheck-v{SHELLCHECK_VERSION}.{postfix}'
    )

    assert 'https://github.com/koalaman/shellcheck/releases/download/v0.9.0/shellcheck-v0.9.0.linux' == url

```

Reply:
that's why I am saying it's a do-nothing PR

you're wasting my time and probably others with similar useless PRs